assets/translate.js

```diff
@@ -124,6 +142,11 @@ $( window ).on( 'load', function () {
 					}
 				} );
 			};
+
+		if ( targetLangCode !== 'fallback' ) {
+			inputWiget.setDisabled( false );
```

Comment:
Are we disabling inputs if the language is the fallback, or if the dropdown shows the "select language" ? it's not very clear from this condition, so I want to make sure.

Could an SVG not have a default language? (could it have only languages inside the switch?) if so, the inputs may be disabled accidentally. Do we want to make this condition be based on what the initial UI value is, instead? 

Either way, please add a quick comment in here; if I'm not 100% sure now, I imagine it won't be completely clear when we get back to it in a year :)